pkg/pillar/types/diskmetrics.go

```diff
@@ -0,0 +1,181 @@
+/*
+ * Copyright (c) 2020. Zededa, Inc.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package types
+
+import (
+	"github.com/lf-edge/eve/pkg/pillar/base"
+	"strings"
+)
+
+//ReportDiskPaths Report disk usage for these paths
+var ReportDiskPaths = []string{
+	"/",
+	IdentityDirname,
+	PersistDir,
+}
+
+//ReportDirPaths  Report directory usage for these paths
+var ReportDirPaths = []string{
+	PersistDir + "/downloads", // XXX old to be removed
+	PersistDir + "/img",       // XXX old to be removed
+	PersistDir + "/tmp",
+	PersistDir + "/log",
+	PersistDir + "/rsyslog",
+	PersistDir + "/config",
+	PersistDir + "/status",
+	PersistDir + "/certs",
+	PersistDir + "/checkpoint",
+}
+
+//AppPersistPaths  Application-related files live here
+// XXX do we need to exclude the ContentTrees used for eve image update?
+// If so how do we tell them apart
+var AppPersistPaths = []string{
+	VolumeEncryptedDirName,
+	VolumeClearDirName,
+	SealedDirName + "/downloader",
+	SealedDirName + "/verifier",
+}
+
+//DiskMetric holds metrics data per disk
+type DiskMetric struct {
+	DiskPath   string
+	ReadBytes  uint64
+	WriteBytes uint64
+	ReadCount  uint64
+	WriteCount uint64
+	TotalBytes uint64
+	UsedBytes  uint64
+	FreeBytes  uint64
```

Comment:
Please add comment to specify the units for these; bytes vs. MBytes. Note that the protobuf fields are in MBytes as specified in the comment in the proto file.

Reply:
Added.